package.json

```diff
@@ -1,17 +1,20 @@
 {
   "name": "babel-plugin-react-cssmoduleify",
-  "version": "0.9.5",
+  "version": "1.0.0-beta",
   "description": "Babel plugin to transform traditional classNames to CSS Modules",
   "main": "lib/index.js",
-  "jsnext:main": "index.js",
+  "jsnext:main": "src/index.js",
```

Comment:
fwiw only [rollup](https://github.com/rollup/rollup/wiki/jsnext:main) uses it as far as I know...though I also can’t use that file directly because of the flow annotations.